### PR TITLE
Add aliases for openstack-k8s-operators team

### PR DESCRIPTION
**Pull request by viroel**
This patch also updates all OWNER files under openstack-k8s-operators directory on step-registry.

### CODE REVIEW

File: clusters/app.ci/registry-access/openstack-k8s-operators/ONWERS [hunk 1]
```diff
@@ -1,14 +1,4 @@
 approvers:
-- abays
```
Comment (marios) [inline clusters/app.ci/registry-access/openstack-k8s-operators/ONWERS:2]: this one has typo in filename onwers

**Review by marios — COMMENTED** [review]
looks ok but inline comment about (pre-existing) typo maybe can be fixed here or you can post a followup

**Comment by arxcruz** [comment]
Regarding to the tempest stuff, this patch removes my name, is it possible to add again?

File: clusters/app.ci/registry-access/openstack-k8s-operators/ONWERS (hunk 1 above)
Comment (viroel) [reply clusters/app.ci/registry-access/openstack-k8s-operators/ONWERS:2]: Yeah, good catch.

File: ci-operator/step-registry/openstack-k8s-operators/tempest/OWNERS [hunk 2]
```diff
@@ -1,17 +1,4 @@
 approvers:
-- abays
-- arxcruz
-- dprince
-- olliewalsh
-- Sandeepyadav93
-- stuggi
-- viroel
-options: {}
+- openstack-k8s-operators-approvers
```
[the remark this answers is not in the document]
Comment (viroel) [reply ci-operator/step-registry/openstack-k8s-operators/tempest/OWNERS:2]: Sorry Arx, added you under aliases.

**Comment by viroel** [comment]
/hold cancel

Fixes the issues pointed earlier

**Pull request merged**